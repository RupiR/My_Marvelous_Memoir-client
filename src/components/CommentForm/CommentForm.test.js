import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import CommentForm from './CommentForm';

describe('<CommentForm />', () => {
    it('Renders without crashing', () => {
        shallow(<CommentForm />);
    });

    it('renders correctly', () => {
        const tree = renderer
            .create(<CommentForm />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});