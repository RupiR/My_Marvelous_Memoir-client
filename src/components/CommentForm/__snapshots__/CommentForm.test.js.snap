// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentForm /> renders correctly 1`] = `
<form
  className="CommentForm"
  onSubmit={[Function]}
>
  <div
    className="text"
  >
    <textarea
      aria-label="Type a comment..."
      className="Textarea "
      cols="30"
      id="text"
      name="text"
      placeholder="Type a comment.."
      required={true}
      rows="3"
    />
  </div>
  <button
    className="Button "
    type="submit"
  >
    Post comment
  </button>
</form>
`;
